### DISCUSSION TITLE
Lost downloads

### DISCUSSION THREAD

**Discussion by wimswims** [transmission/transmission#5908] [Support]
Hi, I hope this is the place for my question.
After starting Transmission Mac again, ALL completed downloads are lost.
So Transmission starts to download all torrents again.
Verify will help some torrents.
What is wrong??? And what can I do to get the completed torrents back???

**Comment by Pentaphon** [comment]
First of all, which Transmission version are you using? What mac version?

**Reply by wimswims** [reply]
I use Transmission 2.93 (because this is the only version my torrent site supports).
On my Mac I use Monterey 12.6.3.
I really hope you can help me...

**Reply by wimswims** [reply]
What I see is that Transmission picked up an old list of torrents. The actual list seems to be gone. Where can I find this lists?
